Source/WTF/wtf/win/FileSystemWin.cpp

```diff
@@ -224,7 +224,7 @@ static String generateTemporaryPath(const Function<bool(const String&)>& action)
 
         ASSERT(wcslen(tempFile) == std::size(tempFile) - 1);
 
-        proposedPath = pathByAppendingComponent(String(tempPath), String(tempFile));
+        proposedPath = pathByAppendingComponent(String(std::data(tempPath)), String(std::data(tempFile)));
```

Comment:
Using std::data doesn't look correct to me. It makes me feeling something wrong.

Reply:
I came up with a good idea to solve this problem. Can I take over this task?

Reply:
I spun off the constructor deletion and relevant fixes to #9282. You can continue there.

`std::data()` just converts the fixed-length array value into the corresponding pointer value, avoiding the deleted constructors.